src/main/java/org/openrewrite/kotlin/internal/KotlinTreeParserVisitor.java

```diff
@@ -1669,7 +1668,7 @@ public J visitKtFile(KtFile file, ExecutionContext data) {
                         Markers.EMPTY,
                         new J.Unknown.Source(
                                 randomId(),
-                                EMPTY,
+                                Space.EMPTY,
```

Comment:
I'm just curious about this change since other parsers typically use `import static org.openrewrite.java.tree.Space.EMPTY;`. Has that changed, or was this an aesthetic preference?

Reply:
yeah, just aesthetic preference, since I think it will make it more clear to differentiate with other `EMPTY`.